pkg/sources/github/github.go

```diff
@@ -695,7 +699,13 @@ func (s *Source) addAllVisibleOrgs(ctx context.Context, apiClient *github.Client
 			log.WithError(err).Errorf("Could not list all organizations")
 			return
 		}
-		s.log.Debugf("listed organization page %d/%d", orgOpts.Page, resp.LastPage)
+		if len(orgs) == 0 {
+			break
+		}
+		lastOrgID := *orgs[len(orgs)-1].ID
+		orgOpts.Since = lastOrgID
+		s.log.Debugf("listed organization IDs %d through %d", orgOpts.Since, lastOrgID)
```

Comment:
`orgOpts.Since` was set to `lastOrgID` in the previous line so this will read "listed organization IDs X through X". Looks like moving it up a line should get the results you want.

Reply:
Thanks Bill!